NuGet.config

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
-      <!-- This feed is used for customized packages which are not published on Nuget.org, and other upstream feeds -->
     <add key="OrchardCore" value="https://www.myget.org/F/orchardcore-dev/api/v3/index.json" />
+    <add key="SixLabors.ImageSharp" value="https://www.myget.org/F/sixlabors/api/v3/index.json" />
```

Comment:
I can add a reference to this feed instead in orchardcore-dev. This way we won't need to do this in code. And when this is merged I will create an issue tagged for beta2 to ensure that the library is in Nuget.org instead. This is a requirement for our public releases, and for beta1 we just took the specific version and put it there using a different name.

Reply:
Done, can you try to remove this line, clear your nuget cache and see if it still restores correctly?

Reply:
Yep, good idea. It works fine!